app/routes/financial.js

```diff
@@ -91,6 +91,21 @@ const FinancialRoute = function(app, db, logger) {
     controller.getData(_key);
   });
 
+  app.put( "/financial/:key", jsonParser, ( req, res, next ) => {
+    if ( !( "value" in req.body ) ) {
+      res.statusCode = 400;
+      next( new Error( "Missing PUT data" ) );
+
+      return;
+    }
+
+    _res = res;
+    _next = next;
+    _key = req.params.key;
+
```

Comment:
@tejohnso They end up being used [here](https://github.com/Rise-Vision/rise-cache-v2/blob/b761341c9807fa8fe36f675f184b2fb0a7dfc2d5/app/routes/financial.js#L40) or [here](https://github.com/Rise-Vision/rise-cache-v2/blob/b761341c9807fa8fe36f675f184b2fb0a7dfc2d5/app/routes/financial.js#L45) depending on which event the controller emits after saving.